src/modules/core/components/InfoPopover/UserInfoPopover.tsx

```diff
@@ -2,19 +2,23 @@ import React from 'react';
 
 import { AnyUser } from '~data/index';
 
-import styles from './InfoPopover.css';
 import UserInfo from './UserInfo';
+import UserInfoNotAvailable from './UserInfoNotAvailable';
+
+import styles from './InfoPopover.css';
 
 interface Props {
-  user: AnyUser;
+  user?: AnyUser;
+  userNotAvailable?: boolean;
 }
 
 const displayName = 'InfoPopover.UserInfoPopover';
 
-const UserInfoPopover = ({ user }: Props) => (
+const UserInfoPopover = ({ user, userNotAvailable = false }: Props) => (
   <div className={styles.main}>
     <div className={styles.section}>
-      <UserInfo user={user} />
+      {userNotAvailable && <UserInfoNotAvailable />}
+      {!userNotAvailable && user && <UserInfo user={user} />}
```

Comment:
{!userNotAvailable && user ? <UserInfo user={user} /> : <UserInfoNotAvailable />
Just to handle weird edgecase when userNotAvailable is false but user = null

Reply:
Nice catch. I've made the change.

Also, you can either format your code suggestions manually using a [code block](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting), or you can use Github's built-in suggestion feature: 
![Screenshot from 2020-11-12 19-59-15](https://user-images.githubusercontent.com/1193222/98977709-95110080-2521-11eb-9ea9-230cc516d84f.png)